roles/common/tasks/monitoring.yml

```diff
@@ -225,3 +225,7 @@
     check: "{{ sensu_checks.common.check_raid }}"
   notify: restart sensu-client missing ok
   when: common.hwraid.enabled|bool
+
+- name: setup vyatta snmp checks on controllers
```

Comment:
I figured adding a name couldn't hurt, but I can remove it for conformity's sake.

As far as a block of tasks, I believe the reason we decided to separate it out of monitoring.yml into its own file is because across all of the teams' requests, we foresaw this list growing considerably over time.